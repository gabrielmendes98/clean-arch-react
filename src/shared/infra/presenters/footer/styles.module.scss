@import 'shared/infra/styles/theme.scss';

.footer {
  background-color: $paper-color;
  color: $primary-color;
}
