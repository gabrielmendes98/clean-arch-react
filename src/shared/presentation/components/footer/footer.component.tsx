import styles from './styles.module.scss';

export const Footer = () => (
  <footer className={styles.footer}>
    &copy; Copyright 2022, Gabriel Santiago
  </footer>
);
