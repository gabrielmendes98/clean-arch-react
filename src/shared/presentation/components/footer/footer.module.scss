@import 'shared/presentation/styles/theme.scss';

.footer {
  background-color: $paper-color;
  color: $primary-color;
}
