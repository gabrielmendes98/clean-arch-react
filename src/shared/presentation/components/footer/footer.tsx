import style from './footer.module.scss';

export const Footer = () => (
  <footer className={style.footer}>
    &copy; Copyright 2022, Gabriel Santiago
  </footer>
);
