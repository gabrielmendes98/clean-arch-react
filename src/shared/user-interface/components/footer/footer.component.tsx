import { memo } from 'react';
import styles from './footer.module.scss';

export const FooterComponent = () => (
  <footer className={styles.footer}>
    &copy; Copyright 2022, Gabriel Santiago
  </footer>
);

export const Footer = memo(FooterComponent);
