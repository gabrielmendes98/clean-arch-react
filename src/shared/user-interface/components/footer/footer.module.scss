@import 'shared/user-interface/styles/theme.scss';

.footer {
  background-color: $paper-color;
  color: $primary-color;
}
