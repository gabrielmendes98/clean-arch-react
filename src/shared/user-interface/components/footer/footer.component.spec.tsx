import { render } from 'shared/testing/test-utils';
import { Footer } from './footer.component';

describe('Footer', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
