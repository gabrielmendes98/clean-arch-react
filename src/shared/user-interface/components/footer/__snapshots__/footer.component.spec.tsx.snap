// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    Â© Copyright 2022, Gabriel Santiago
  </footer>
</DocumentFragment>
`;
